pandas/compat/_optional.py

```diff
@@ -33,6 +33,32 @@
     "numba": "0.46.0",
 }
 
+INSTALL_MAPPING = {
```

Comment:
only add the names of packages that are *different* (as we already have all of the names); add a comment to describe this.

Reply:
I have made the changes as suggested.